frontend/app/components/Notification.tsx

```
@@ -1,15 +1,15 @@
-      </p>
-    </div>
-  );
-}
+export default function Notifications({}: {}) {
+  return (
+    <div className="w-full bg-gray-50/5 text-sm mt-20 text-white shadow-lg px-2 py-1.5 text-center z-50">
+      <p className="">
+        🎉 WR.DO Beta Launching Now!{" "}
+        <a
+          target="_blank"
+          className="text-blue-500 ml-2 underline after:content-['_↗']"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)